src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,89 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { PropsWithChildren, ReactElement, RefObject, useEffect, useState } from 'react';
+import { usePopper } from 'react-popper';
+import { merge } from '@utilities/merge';
+import { MenuItemProps } from '@components/MenuItem';
+import { useMenuKeyboardNavigation } from '@components/Menu/useMenuKeyboardNavigation';
+interface Props {
+    triggerRef?: RefObject<Element>;
+    open?: boolean;
+    children?: ReactElement<MenuItemProps> | ReactElement<MenuItemProps>[];
+    onClose?: () => void;
+}
+
+export type MenuProps = PropsWithChildren<Props>;
+
+const CONTAINER_BASE_CLASSES = 'tw-relative tw-bg-base tw-rounded tw-py-2 tw-shadow-mid tw-w-max';
+const CONTAINER_POSITION_CLASSES = ' tw-left-0 tw-top-7';
```

Comment:
Is the at the whitespace needed in the beginning of the string? Also, is this menu supposed to be used with any trigger? The `top: 7` doesn't allow that, but maybe I'm misunderstanding here :)

Reply:
You are right, I will remove that because popper is positioning the element. And the whitespace shouldn't be there at all. Thanks for the catch.

Reply:
Actually, we don't need the `CONTAINER_POSITION_CLASSES` anymore. I will just remove it. That's why I like so separate TW classes in some variables. It is too hard to catch those cases when we are dealing with a super long string.